### PR TITLE
Created Dictionary Pair Programming Post

### PR DESCRIPTION
**Pull request by ghost**
Created first draft of Kevin and Audrey's dictionary exercises pair programming reflections. Will add reflection after it is successfully set up as a co-authored post!

### CODE REVIEW

File: _posts/intjnotinfp/2020-10-20-dictionary-pair-exercise-kevin-audrey [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+author:
```
Comment (MishaCienfuegos) [inline _posts/intjnotinfp/2020-10-20-dictionary-pair-exercise-kevin-audrey]: Looks like this is missing the `layout: post`.

**Review by MishaCienfuegos — CHANGES_REQUESTED** [review]
This is looking good! It does appear the file is missing the `.md`. Could you update that?

**Review by KJEZ35 — APPROVED** [review]
Went back to add the .md

File: _posts/intjnotinfp/2020-10-20-dictionary-pair-exercise-kevin-audrey (hunk 1 above)
Comment (KJEZ35) [reply _posts/intjnotinfp/2020-10-20-dictionary-pair-exercise-kevin-audrey]: @MishaCienfuegos went back and fixed this, thank you!

**Comment by eah13** [comment]
@KJEZ35 Nice work on the file extension. 

The post is still missing a `layout: post` in the YAML header (see comment from Misha above). Your comment there says you addressed this, but I don't see that in the code. One more quick fix and this is ready for merge

**Review by KJEZ35 — APPROVED** [review]
Added the layout, 11-06-2020

File: _posts/intjnotinfp/2020-10-20-dictionary-pair-exercise-kevin-audrey (hunk 1 above)
Comment (KJEZ35) [reply _posts/intjnotinfp/2020-10-20-dictionary-pair-exercise-kevin-audrey]: Thank you for the catch @MishaCienfuegos, went back and added this.

**Review by MishaCienfuegos — APPROVED** [review]
Awesome, looks good! 🍂

**Pull request merged**